frontend/src/pages/secret-manager/SecretDashboardPage/components/SecretListView/SecretItem.tsx

```diff
@@ -1,5 +1,24 @@
-/* eslint-disable no-nested-ternary */
-/* eslint-disable simple-import-sort/imports */
+import { memo, useCallback, useEffect, useRef } from "react";
```

Comment:
Unfortunately this component had a lot of logic mixed up between rendering a personal override and working with a secret as usual.

Please review and test, functionality of handling personal overrides has been stripped, it on creates and leaves the presentaiton and other ops to `SecretPersonalOverrideView`